doc/kubernetes/modules/ROOT/pages/running/network-partition.adoc

```diff
@@ -0,0 +1,112 @@
+= Out-of-sync datacenters Recovery
+:description: This guide describes the steps to bring the secondary site up to date to the primary site after a network partition.
+
+include::partial$running/infinispan-attributes.adoc[]
+
+{description}
+
+// Infinispan -> clear caches on secondary site and perform the state transfer
+// Keycloak -> clear caches (how?)
+// Aurora -> ?
+// Route53 -> no-op
+
+== Audience
+
+This guide describes the procedures required to synchronize the secondary site after a temporary disconnection between sites.
+
+See xref:running/index.adoc[] for additional guides.
+
+=== {ispn} Cluster
+
+For the context of this guide, `{site-a}` is the primary site and `{site-b}` is the secondary site.
+
+Network partitions may happen between the site and the replication between the {ispn} cluster will stop.
+Manual steps described in this guide bring both site back in sync.
+
+include::partial$running/infinispan-batchcr-intro.adoc[]
+
+The `ConfigMap` with the {ispn} CLI operations are as follows:
+
+.Checks the cross-site status for all caches backing up to `{site-b}`.
+[source,yaml]
+----
+include::example$helm/ispn-site-a.yaml[tag=infinispan-crossdc-status]
+----
+
+.Starts a cross-site state transfer for all caches backing up to `{site-b}`.
+[source,yaml]
+----
+include::example$helm/ispn-site-a.yaml[tag=infinispan-crossdc-push-state]
+----
+
+.Checks the cross-site state transfer progress.
+[source,yaml]
+----
+include::example$helm/ispn-site-a.yaml[tag=infinispan-crossdc-push-state-status]
+----
+
+.Resets the cross-site state transfer progress after its completion.
+[source,yaml]
+----
+include::example$helm/ispn-site-a.yaml[tag=infinispan-crossdc-reset-push-state-status]
+----
+
+Install those `ConfigMap` in the primary site, `{site-a}`.
+
+The caches in the secondary site contain stale data, and must be clear before receiving state.
+Install the following `ConfigMap` into the secondary site, `{site-b}`:
+
+.Clears the cache data in `{site-b}`.
+[source,yaml]
+----
+include::example$helm/ispn-site-b.yaml[tag=infinispan-crossdc-clear-caches]
+----
+
+[#running-infinispan-batch]
+==== Running {ispn} Batch
+
+include::partial$running/infinispan-running-batch.adoc[]
+
+==== Procedures
+
+WARNING: Transferring the full state may impact the {ispn} cluster perform by increasing the response time and/or resources usage.
+
+First procedure is to delete the stale data from the secondary site.
+
+. Login into your secondary site.
+
+. Shutdown Keycloak.
```

Comment:
How should the user do this?

Reply:
I will add that they should change the number of instances to 0 in the CR.